src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -228,7 +228,19 @@ public PersonBuilder withSymptom(String symptom) {
         return this;
     }
 
+    /**
+     * Returns a Person object with fields initialised to that of PersonBuilder object.
+     */
     public Person build() {
-        return new Person(nric, name, phone, address, dateOfBirth, sex, status);
+        Person p = new Person(nric, name, phone, address, dateOfBirth, sex, status);
+        p.setEmail(email);
```

Comment:
not sure if this will cause conflicts with JAP (JsonAdaptedPerson), will check later

Reply:
didn't find conflicts, go ahead